bigquery/google/cloud/bigquery/_helpers.py

```diff
@@ -224,21 +224,44 @@ def _row_tuple_from_json(row, schema):
 
     Args:
         row (Dict): A JSON response row to be converted.
-        schema (Tuple): A tuple of :class:`~google.cloud.bigquery.schema.SchemaField`.
+        schema (Sequence[Union[ \
+            Sequence[:class:`~google.cloud.bigquery.schema.SchemaField`], \
```

Comment:
I think we have a few too many `Sequences` here. It's either a Union of Sequences or a Sequence of Unions.

I was originally thinking a Union of Sequences, but now that I've seen Tres's suggestion below, I think a Sequence of Unions wouldn't may be simpler for implementation.

Ditto elsewhere the `schema` argument is used.

Reply:
Let's go with `Sequence[Union[...]]` then, and use the more straightforward implementation as Tres suggested. :+1: